_themes/sphinx_rtd_theme/moveit_version.html

```diff
@@ -1,6 +1,5 @@
 <br />
 <div class="admonition note">
-<p class="first admonition-title">Tutorials Version: Kinetic</p>
-<p class="last">This is the ROS Kinetic version of the tutorials, intended for ROS Kinetic users only.<br/>
-If you are using a ROS Melodic release, we recommmend the <a class="reference external" href="http://docs.ros.org/melodic/api/moveit_tutorials/html/index.html">Melodic tutorials</a> or the latest <a class="reference external" href="https://ros-planning.github.io/moveit_tutorials/">master branch tutorials</a>.
+<p class="first admonition-title">WARNING: Old Tutorial Version</p>
+<p class="last">The latest (and last) version of <a class="reference external" href="http://ros-planning.github.io/moveit_tutorials/">MoveIt 1 is for ROS Noetic</a>, which is still actively developed. See also <a class="reference external" href="http://moveit2_tutorials.picknik.ai/">MoveIt 2 tutorials</a> and other available versions in drop down box on left.</p
```

Comment:
```suggestion
<p class="last">
   The latest (and last) version of <a class="reference external" href="http://ros-planning.github.io/moveit_tutorials/">MoveIt 1 is for ROS Noetic</a>, which is still actively developed. 
   See also <a class="reference external" href="http://moveit2_tutorials.picknik.ai/">MoveIt 2 tutorials</a> and other available versions in drop down box on left.
   </p>
```
was missing end bracket